portfolio/src/main/webapp/script.js

```diff
@@ -16,13 +16,40 @@
  * Adds a random greeting to the page.
  */
 function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+  //const greetings =
+  //    ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
 
+  const imagesArray = 
+  ['images/youtube_rewind.png','images/chrome_ram.gif', 'images/coffee.png','images/sundar.png'];
   // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
+  var i;
```

Comment:
Instead of having this line out here, it would be more clear if it was "inlined" into the for-loop below, so below it would say:

`for (var i = 0; i < imagesArray.length; i++)`